### ISSUE TITLE
When `type` is given `"Text"` it generates undesirable results

### ISSUE THREAD

**Issue by Farcrada** [rauenzi/BDPluginLibrary#92]
https://github.com/rauenzi/BDPluginLibrary/blob/105f02353cbe2b0f6641aac2fa45486f3e09e849/src/ui/discordcontextmenu.js#L161-L181

When the `type` property is filled with `Text` (see: https://github.com/Echological/MentionChannels-BD/issues/2) it becomes a group menu?

The original assigning here in PermissionViewer: https://github.com/rauenzi/BetterDiscordAddons/blob/38cd17c7e8dad92775aa754d493755b6a119585b/Plugins/PermissionsViewer/PermissionsViewer.plugin.js#L571, might also play a hand. However when we remove the `type` property from MentionChannels it works like a charm.

I had an issue with this (more here: https://github.com/Farcrada/DiscordPlugins/issues/47), where my context item would show up inside the MentionChannels button. When `type: "Text"` property is removed from MentionChannels' patch(es) it's everything starts working as intended again.

I hope this information will help.

**Comment by Farcrada** [comment]
Oh, now I am confused. I have changed the way I load/position/render my context item: https://github.com/Farcrada/DiscordPlugins/blob/master/Move-All-Voice-Users/MoveAllVoiceUsers.plugin.js#L127-L131 so now I've worked around the issue. And PermissionViewer still has the issue, but I am unsure if the issue is with the way PermissionViewer adds it's context item or if there's something amiss entirely.
